SimCalorimetry/HGCalSimProducers/python/hgcalDigitizer_cfi.py

```diff
@@ -233,19 +155,18 @@
         xTalk     = cms.double(0.01),
         sdPixels  = cms.double(1e-6), # this is additional photostatistics noise (as implemented), not sure why it's here...
         thresholdFollowsMIP = cms.bool(thresholdTracksMIP),
-        feCfg   = cms.PSet(
-            # 0 only ADC, 1 ADC with pulse shape, 2 ADC+TDC with pulse shape
-            fwVersion       = cms.uint32(0),
-            # n bits for the ADC (same as the silicon ROC)
-            adcNbits        = cms.uint32(12),
-            # ADC saturation : in this case we use the same variable but fC=MIP
-            adcSaturation_fC = cms.double(275.0), #value chosen to have 1MIP at 15ADC,
-            # together with the choice of adcNbits
-            # threshold for digi production : in this case we use the same variable but fC=MIP
-            adcThreshold_fC = cms.double(0.5),
-            )
+        feCfg = hgcROCSettings.clone(
+            adcNbits        = cms.uint32(10),      # standard ROC operations (was 2 bits more up to 11_0_0_pre12)
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
addressed.